app/models/setting.rb

```diff
@@ -264,42 +261,19 @@ def self.bypass_readonly(s, &block)
     s.readonly! if old_readonly
   end
 
-  def self.cache
-    Rails.cache
-  end
-
   # Methods for loading default settings
 
   def self.default_settings
     []
   end
 
   def self.load_defaults
-    return false unless table_exists?
-    dbcache = Hash[Setting.where(:category => name).map { |s| [s.name, s] }]
-    transaction do
-      default_settings.compact.each do |s|
-        val = s.update(:category => name).symbolize_keys
-        dbcache.key?(val[:name]) ? create_existing(dbcache[val[:name]], s) : create!(s)
-      end
-    end
-    true
-  end
-
-  def self.select_collection_registry
-    @@select_collection ||= SettingSelectCollection.new
+    nil # Already loaded
```

Comment:
If settings are already loaded, do we still need this method? I do not see it being called anywhere...

Reply:
`Katello` is calling it explicitly. But we should probably deprecate it :thinking: